packages/terra-dropdown-button/tests/wdio/dropdown-button-spec.js

```diff
@@ -109,19 +109,19 @@ Terra.describeViewports('Dropdown Button', ['medium'], () => {
         Terra.validates.element('click');
       });
 
-      it('keyboard navigates down past the last option', () => {
-        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Space']);
-        Terra.validates.element('no down out of bounds');
+      it('Navigates from last item to first item', () => {
+        browser.keys(['ArrowDown', 'ArrowDown', 'ArrowDown', 'ArrowDown', 'Enter']);
+        Terra.validates.element('first item focused');
```

Comment:
I still see old screenshots `no down out of bounds` for both dropdown-button on master repo which are supposed to be removed with this update

Reply:
Removed old screenshots.